tests/e2e/core-tests/specs/shopper/front-end-cart-calculate-shipping.test.js

```diff
@@ -43,29 +39,19 @@ const runCartCalculateShippingTest = () => {
 			await createSimpleProduct(firstProductName);
 			await createSimpleProduct(secondProductName, secondProductPrice);
 
-			await merchant.login();
-			await merchant.openNewShipping();
-
 			// Add a new shipping zone Germany with Free shipping
```

Comment:
I got the tests to pass by adding a reset before adding the shipping zones

```
			await withRestApi.resetSettingsGroupToDefault( 'general' );
```

```
Test Suites: 1 skipped, 38 passed, 38 of 39 total
Tests:       2 skipped, 164 passed, 166 total
Snapshots:   0 total
Time:        752.343s
Ran all test suites.
Jest exit code: 0
```

Reply:
Oh good idea, I just added that in and they ran successfully for me, too:

```
Test Suites: 1 skipped, 38 passed, 38 of 39 total
Tests:       2 skipped, 164 passed, 166 total
Snapshots:   0 total
Time:        791.236s
Ran all test suites.
Jest exit code: 0
```

Pushed up changes including that line